src/Header/Header.js

```diff
@@ -6,22 +6,25 @@ import { Navbar, Nav, Button } from 'react-bootstrap';
 const Header = () => {
   return (
     <header className="navHeader">
-      <Navbar expand="lg" class="d-flex flex-column flex-md-row align-items-center px-3 pt-2 px-md-4 mb-3 bg-white border-bottom shadow-sm">
-      <Navbar.Brand href="#home">
-          <img src={logo} class="logo" alt="Reclass logo"/>
-      </Navbar.Brand>
-      <Navbar.Toggle aria-controls="basic-navbar-nav" />
-      <Navbar.Collapse id="basic-navbar-nav">
+      <Navbar
+        expand="lg"
+        class="d-flex flex-column flex-md-row align-items-center px-3 pt-2 px-md-4 mb-3 bg-white border-bottom shadow-sm"
```

Comment:
I think this is supposed to be `className` instead of `class`

Reply:
Yes, I think so as well, this was Yi's code when he started working on the navbar; I'll clean this up!